### DISCUSSION TITLE
RX and TX simultaneously

### DISCUSSION THREAD

**Discussion by maxlacourchristensen** [svpcom/wfb-ng#275] [Questions]
Hello, 

I am using RPI4B and Asus AC56.

I am curious if it is possible to send and receive packets at the same time without significant increase in dropped packets? I have experienced that if the base station sends packets at the same time as the drone transmits packets, I get a significant increase in dropped packets. Is it possible to use a NIC with more antennas and somehow subdivide the band (20/40 MHz) to allow one antenna to do RX and another to do TX at the same time? I have another RTL8812AU NIC with 2 external antennas.

Do you know if there is anything related to MIMO or OFDM which can help in this matter?

**Comment by svpcom** [comment]
You can setup two different nics (each on separate channel) for tx and rx. You need to configure two instances of wfb-ng or modify python code to use different nics for tx and rx and different frequency channels for them
